content/docs/command-reference/remote/add.md

```diff
@@ -482,3 +483,21 @@ $ dvc remote list
 
 myremote	s3://mybucket/another-path
 ```
+
+The credential information can be set via environment variables, for example:
+
+```dvc
+$ export AWS_ACCESS_KEY_ID="<my-access-key>"
```

Comment:
This example is about customizing the remote (in .dvc/config) so this new text would maybe need to be it's own separate example?

Reply:
But the info about setting up credentials is already in the expandable sections. Not sure we need an example here. Is this trying to solve a specific scenario?